LInjector/LInjector/Classes/MonacoInterfacer.cs

```diff
@@ -1,117 +1,173 @@
         {
             EventHandler handler = EditorReady;
             if (handler != null)
                 handler(this, new EventArgs());
-
         }
 
-        public void monaco_api_CoreWebView2InitializationCompleted(object sender, Microsoft.Web.WebView2.Core.CoreWebView2InitializationCompletedEventArgs e)
+        public void MonacoApi_CoreWebView2InitializationCompleted(object sender, Microsoft.Web.WebView2.Core.CoreWebView2InitializationCompletedEventArgs e)
         {
             this.CoreWebView2.DOMContentLoaded += CoreWebView2_DOMContentLoaded;
             this.CoreWebView2.WebMessageReceived += CoreWebView2_WebMessageReceived;
             this.CoreWebView2.Settings.AreDefaultContextMenusEnabled = false;
             this.CoreWebView2.Settings.AreDevToolsEnabled = false;
         }
 
-        private void CoreWebView2_WebMessageReceived(object sender, Microsoft.Web.WebView2.Core.CoreWebView2WebMessageReceivedEventArgs e) => LatestRecievedText = e.TryGetWebMessageAsString();
+        private void CoreWebView2_WebMessageReceived(object sender, Microsoft.Web.WebView2.Core.CoreWebView2WebMessageReceivedEventArgs e) => LatestReceivedText = e.TryGetWebMessageAsString();
 
         private async void CoreWebView2_DOMContentLoaded(object sender, Microsoft.Web.WebView2.Core.CoreWebView2DOMContentLoadedEventArgs e)
         {
             await Task.Delay(1000);
-            isDOMLoaded = true;
+            IsDOMLoaded = true;
             SetText(ToSetText);
             OnEditorReady();
         }
+
+        /// <summary>
+        /// Retrieves the text content from the Monaco Editor.
+        /// </summary>
+        /// <returns>The text content from the editor.</returns>
         public async Task<string> GetText()
         {
-            if (isDOMLoaded)
+            if (IsDOMLoaded)
             {
                 var script = "monaco.editor.getModels()[0].getValue()";
-                var result = await CoreWebView2.ExecuteScriptAsync(script);
+                var result = await this.ExecuteScriptAsync(script);
                 var text = JsonConvert.DeserializeObject<string>(result);
 
                 return text;
             }
             return string.Empty;
         }
+
+        /// <summary>
+        /// Sets the text content of the Monaco Editor.
+        /// </summary>
+        /// <param name="text">The text to set in the editor.</param>
         public async void SetText(string text)
         {
-            if (isDOMLoaded)
+            if (IsDOMLoaded && CoreWebView2 != null)
+            {
                 text = text.Replace("\\", "\\\\");
-            await CoreWebView2.ExecuteScriptAsync("editor.setValue('');");
-            await CoreWebView2.ExecuteScriptAsync($"editor.setValue(`{text.Replace("`", "\\`")}`)");
+                await this.ExecuteScriptAsync("editor.setValue('');");
+                await this.ExecuteScriptAsync($"editor.setValue(`{text.Replace("`", "\\`")}`)");

```

Comment:
Using `this` instead of `CoreWebView2` because the local method validates the integrity of the WebView. I also think this should be the preferred method in this code.